packages/svelte/src/internal/client/reactivity/effects.js

```diff
@@ -278,7 +289,16 @@ export function legacy_pre_effect(deps, fn) {
 
 		token.ran = true;
 		set(context.l.r2, true);
-		untrack(fn);
+
+		if (location) {
```

Comment:
better safe than sorry

```suggestion
		if (DEV && location) {
```